tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of IDUtils
+# Copyright (C) 2024 CERN.
+#
+# IDUtils is free software; you can redistribute it and/or modify
+# it under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Pytest configuration."""
+
+import pytest
+from invenio_app.factory import create_api
+
+
+@pytest.fixture(scope="session")
+def create_app():
+    """Application factory fixture."""
+    return create_api
```

Comment:
I think we miss some tests about the new entrypoint functionalities. Basically, we should test the different entrypoints registration and showcase examples of callable functions that are expected there.